app/src/main/java/com/duckduckgo/app/browser/BrowserTrackersAnimatorHelper.kt

```diff
@@ -234,6 +234,7 @@ class BrowserTrackersAnimatorHelper {
             .asSequence()
             .distinct()
             .take(MAX_LOGOS_SHOWN + 1)
+            .sortedWith(compareBy { "AEIOU".contains(it.displayName[0]) })
```

Comment:
Just a suggestion. I think at some point we may not remember why this logic was here, and since we don't have tests for this class (document purposes), I suggest to create a method so we can make it more descriptive.

We could just create a extension function for `Sequence<Entity>`, it will look like this:
```
    private fun Sequence<Entity>.sortToAvoidDodgyWords(): Sequence<Entity> {
        return sortedWith(compareBy { "AEIOU".contains(it.displayName[0]) })
    }
````
and yout code will change in this way:
```
(...)
.take(MAX_LOGOS_SHOWN + 1)
.sortToAvoidDodgyWords()
.map {
(...)
```

Reply:
But we can avoid the use of the word `dodgy`? `sortToAvoidSpellingWords()` perhaps

Reply:
sure, I was just using @brindy 's word as an example :D